src/swapd/runtime.rs

```diff
@@ -109,12 +109,13 @@ pub fn run(
 
     info!("Initial state: {}", init_state.bright_white_bold());
 
+    // FIXME: threshold and race should be parameters
     let temporal_safety = TemporalSafety {
         cancel_timelock: cancel_timelock.as_u32(),
         punish_timelock: punish_timelock.as_u32(),
-        btc_finality_thr: 0,
+        btc_finality_thr: 2,
```

Comment:
I'm against this change, the swap is still not running reliably enough for me to warrant waiting for confirmations. I'd be willing to accept these as default values, once the thresholds are parameterized.

Reply:
Let put this on the side for now then, I'll change it to a draft